python/dataclean.py

```diff
@@ -90,6 +90,34 @@ def addSeason(cursor, seasonsData, qualiResultsData, qualiChanges, year):
     seasonsData[year] = s
     return no_mistakes
 
+def addRaceSeasonData(cursor, seasonsData, raceResultsData, year):
+    """Adds a Season Data object to the given map of seasons"""
+    s = Season()
+
+    sql = "SELECT `raceId`, `round`, `circuitId`, `name` FROM `races` WHERE `year`=%s"
+    cursor.execute(sql, year)
+    result = cursor.fetchall()
+    
+    for x in result:
+        circuitId = x.get('circuitId')
+        roundNo = x.get('round')
+
+        raceData = RaceData(circuitId, roundNo)
+        s.addRace(x.get('raceId'), raceData)
+
```

Comment:
I think there was a misunderstanding with the requested changes. You still need the function call to `addRaceResults()`, just that the assignment to variable `res` was unnecessary. it should be  
`addRaceResults(cursor, raceResultsData, x.get('raceId'))`, not  
`res = addRaceResults(cursor, raceResultsData, x.get('raceId'))`